shuffle-plugin/pom.xml

```diff
@@ -61,7 +61,7 @@
         <dependency>
           <groupId>org.openucx</groupId>
           <artifactId>jucx</artifactId>
-          <version>1.9.0</version>
+          <version>1.11.0-SNAPSHOT</version>
```

Comment:
Lets target the PR for branch-0.6. We are looking to release branch-0.5 soon and are in burn-down mode, so I really don't want to change 0.5 unless we have to.

Also note that changing to 1.11.0-SNAPSHOT is probably fine while we work on the branch, but we'll want a real version once JUCX 1.11 is re-published, and that we should double check https://github.com/NVIDIA/spark-rapids/blob/branch-0.5/NOTICE-binary, to make sure that UCX license is accurate. Seems like it is so far.

Reply:
So fortunately we have 1.11.0 release in maven, that can be used. If there would be critical issues in JUCX we'll release a new one.